src/VisualStudio/Core/Test/ProjectSystemShim/VisualStudioAnalyzerTests.vb

```diff
@@ -29,7 +29,12 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim
 
         <WpfFact, Trait(Traits.Feature, Traits.Features.Diagnostics)>
         Public Sub AnalyzerErrorsAreUpdated()
-            Dim hostDiagnosticUpdateSource = New HostDiagnosticUpdateSource(Nothing, New MockDiagnosticUpdateSourceRegistrationService())
+            Dim lazyWorkspace = New Lazy(Of VisualStudioWorkspaceImpl)(
+                                    Function()
+                                        Return Nothing
```

Comment:
For the sake of tests, should this throw if they ever try to use this? I'm guessing they don't, so maybe better to make the mock explode?